rllib/policy/torch_policy.py

```diff
@@ -39,23 +39,22 @@ class TorchPolicy(Policy):
     """
 
     @DeveloperAPI
-    def __init__(self,
-                 observation_space: gym.spaces.Space,
-                 action_space: gym.spaces.Space,
-                 config: TrainerConfigDict,
-                 *,
-                 model: ModelV2,
-                 loss: Callable[
-                     [Policy, ModelV2, type, SampleBatch], TensorType],
-                 action_distribution_class: TorchDistributionWrapper,
-                 action_sampler_fn: Callable[
-                     [TensorType, List[TensorType]], Tuple[
-                         TensorType, TensorType]] = None,
-                 action_distribution_fn: Optional[Callable[
-                     [Policy, ModelV2, TensorType, TensorType, TensorType],
-                     Tuple[TensorType, type, List[TensorType]]]] = None,
-                 max_seq_len: int = 20,
-                 get_batch_divisibility_req: Optional[int] = None):
+    def __init__(
+            self,
+            observation_space: gym.spaces.Space,
+            action_space: gym.spaces.Space,
+            config: TrainerConfigDict,
+            *,
+            model: ModelV2,
+            loss: Callable[[Policy, ModelV2, type, SampleBatch], TensorType],
+            action_distribution_class: TorchDistributionWrapper,
+            action_sampler_fn: Callable[[TensorType, List[TensorType]], Tuple[
+                TensorType, TensorType]] = None,
+            action_distribution_fn: Optional[Callable[[
+                Policy, ModelV2, TensorType, TensorType, TensorType
+            ], Tuple[TensorType, type, List[TensorType]]]] = None,
+            max_seq_len: int = 20,
+            get_batch_divisibility_req: Optional[int] = None):
```

Comment:
Not sure why lint decided to reformat this, I didn't change anything here.

Reply:
np, actually: This looks nicer :)